tests/php/Domain/Services/DeleteDraftOrders.php

```diff
@@ -61,14 +86,18 @@ public function setUp() {
 	 */
 	public function test_delete_expired_draft_orders() {
 		global $wpdb;
+		$status = DraftOrders::DB_STATUS;
 
 		// Check there are 3 draft orders from our setup before running tests.
-		$this->assertEquals( 3, (int) $wpdb->get_var( "SELECT COUNT(ID) from $wpdb->posts posts WHERE posts.post_status = 'wc-checkout-draft'" ) );
+		$this->assertEquals( 3, (int) $wpdb->get_var( "SELECT COUNT(ID) from $wpdb->posts posts WHERE posts.post_status = '$status'" ) );
```

Comment:
It would be better to use $wpdb->prepare here even though it's in the test suite (for status).